modules/installing/pages/enabling-builds.adoc

```diff
@@ -79,3 +79,10 @@ _Uses `buildah` to create a multi-platform container image leveraging link:https
 This pipeline is ideal for building and verifying xref:end-to-end:building-olm.adoc#building-the-file-based-catalog[file-based catalogs].
 
 _Uses `buildah` to create a container image. Its build-time tests are limited to verifying the included catalog and not to scanning the image itself. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-fbc-builder?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/fbc-builder/README.md[parameter documentation])_
+
+==== _tekton-bundle-builder-oci-ta_ [[tekton-bundle-builder-oci-ta]]
+
+This pipeline is ideal for building Tekton tasks as bundles.
+
+_Uses `tkn-bundle-oci-ta` task to create tekton task bundles and push them as an oci-artifacts to a repository.
+Information is shared between tasks using OCI artifacts instead of link:https://kubernetes.io/docs/concepts/storage/persistent-volumes[Persistent Volume Claims]. EC will pass the link:https://conforma.dev/docs/ec-policies/release_policy.html#trusted_task__trusted[trusted_task.trusted] policy if all data used to build the artifact is generated from trusted tasks. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-tekton-bundle-builder-oci-ta?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/tekton-bundle-builder-oci-ta/README.md[parameter documentation])_
```

Comment:
```suggestion
Information is shared between tasks using OCI artifacts instead of link:https://kubernetes.io/docs/concepts/storage/persistent-volumes[Persistent Volume Claims]. Conforma will pass the link:https://conforma.dev/docs/ec-policies/release_policy.html#trusted_task__trusted[trusted_task.trusted] policy if all data used to build the artifact is generated from trusted tasks. (link:https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-tekton-bundle-builder-oci-ta?tab=tags[quay.io link], link:https://github.com/konflux-ci/build-definitions/blob/main/pipelines/tekton-bundle-builder-oci-ta/README.md[parameter documentation])_
```